app/views.py

```diff
@@ -144,6 +144,17 @@ def player_join():
                 break
     g_random_set.add(random_idx)
 
+    alias = ""
+    profession = ""
+
+    should_include_alias = request.form.getlist('alias')
+    if (should_include_alias):
+        alias = ds.get_alias()
+    
+    should_include_profession = request.form.getlist('profession')
+    if (should_include_profession):
+        profession = ds.get_profession()
+
```

Comment:
@6lor the above change would then involve calling one more function here..

Reply:
You can call the 2 functions and just concatenate the 2 results and send it to `render_template`